API.md

```diff
@@ -316,17 +316,18 @@ A low-level interface to geocoding, useful for more complex uses and reverse-geo
 | Options | Value | Description |
 | ---- | ---- | ---- |
 | id _or_ url | string | Value must be <ul><li>A [geocoder index ID](https://www.mapbox.com/developers/api/geocoding/), e.g. `mapbox.places`</li><li>A geocoder API URL, like `{{site.tileApi}}/v4/geocode/mapbox.places/{query}.json`</li></ul> |
-| options | Object | The second argument is optional. If provided, it may include: <ul><li>`accessToken`: Mapbox API access token. Overrides `L.mapbox.accessToken` for this geocoder.</li></ul> |
+| options | Object | The second argument is optional. If provided, it may include: <ul><li>`accessToken`: Mapbox API access token. Overrides `L.mapbox.accessToken` for this geocoder.</li><li>`proximity`: boolean which enables viewport based proximity search</li></ul> |
 
 _Returns_ a `L.mapbox.geocoder` object.
 
-### geocoder.query(queryString, callback)
+### geocoder.query(queryString|options, callback)
 
 Queries the geocoder with a query string, and returns its result, if any.
 
 | Options | Value | Description |
 | ---- | ---- | ---- |
 | queryString (_required_) | string | a query, expressed as a string, like 'Arkansas' |
+| options | object | an object containing the query and options parameters like `{ query: 'Austin', proximity: [lng, lat] }`
```

Comment:
I think this `proximity` object should be an object like `L.latLng` rather than a `[lng, lat]` array: L.latLng is the Leaflet expectation and it's what we handle elsewhere in this api, like in setView. Leaflet's expectations are the law of the land around here, whether we like latlon or not